### ISSUE TITLE
Comparison view for extension properties hallucinates unsupported properties

### ISSUE THREAD

**Issue by per-mathisen-arm** [SaschaWillems/vulkan.gpuinfo.org#63]
Take a look at https://vulkan.gpuinfo.org/compare.php?reports=17913,17914#properties_extensions - Mali does not support the VK_EXT_blend_operation_advanced, for example, but it is listed here with what looks like imaginary values. You can check that the same device report does not list them at all here: http://vulkan.gpuinfo.org/displayreport.php?id=17913#properties_extensions

**Comment by per-mathisen-arm** [comment]
There is also a discrepancy for https://vulkan.gpuinfo.org/displayreport.php?id=17914#device between the extension list in the web view and the extension list in the linked JSON download. The web view for instance lists many more extensions as supported than the JSON, eg VK_KHR_ray_query is only in the web view.

**Comment by SaschaWillems** [comment]
The JSON is structured based on the LunarG profiles schema. That'll probably need some update to expose newer features.

Thanks for raising those issues, I'll try to fix both of them.

**Comment by SaschaWillems** [comment]
Just checked, and this seems to be a mismatch between the device version and how schema files are mapped. I use the patch level from the api version, which in that case is 128 and use that to look up the LunarG JSON schema file to generate the JSON. But this will select a 1.2 schema file that doesn't know about that extension. Will have to think about a way of solving this.

**Comment by SaschaWillems** [comment]
The wrong value display for the report compare has been fixed. This was a stupid oversight on my side (a variable not getting properly initialized) and affected pretty much all comparisons. It should now work as expected..

I also did some minor changes to the compare pages. The header now also contains the device name (on Android) and some potentially large values are shortened so they don't break the layout.

Again, thanks a lot for bringing this up 👍
